docs/modules/ROOT/partials/cmos-health-checks.adoc

```diff
@@ -195,4 +195,15 @@ Alternatively, navigate to your Data storage directory and enter the following c
 
 *Further Reading*: https://docs.couchbase.com/server/current/learn/data/data.html#metadata[Metadata], https://docs.couchbase.com/server/current/rest-api/rest-bucket-create.html#evictionpolicy[evictionPolicy].
 
+[#CB90061]
+=== Disk Write Queue (CB90061)
+
+*Background*: Having items in the disk write queue for too long could cause backoffs. This usually indicates either a sizing issue, or a storage medium failing.
+
+*Condition*: Age of items in disk write queue over 40 seconds.
```

Comment:
When does this become critical?

Reply:
Should the condition instead read: "Age of items in disk write queue over 40 seconds produces a warning, and over 60 an alert."?

Reply:
Something like that, or perhaps "upgraded to critical if the age is over 60"?